app/Components/Camera/Camera.js

```diff
@@ -7,18 +7,30 @@ import {
   Dimensions
 } from "react-native";
 import Camera from "react-native-camera";
+const RNFS = require("react-native-fs");
```

Comment:
no using anymore.  needs deleting.  or further research about whether I need to be using it